File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/attivita/note_intervento/NoteIntervento.ts
import { BaseCodeDescr } from "app/Model/baseClass/baseCodeDescr";
import { NoteInterventoGruppi } from "./NoteInterventoGruppi";


export class NoteIntervento extends BaseCodeDescr {

    note_Valore_Numerico: number;

    note_Valore_Stringa: string;

    visibile: number;

    noteInterventoGruppi: NoteInterventoGruppi;

    constructor(codice: number) {
        super(codice);
    }
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/attivita/note_intervento/NoteIntervento.cs
using AgronicaCoreModelsSTD.baseClass;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.attivita.note_intervento
{
    public class NoteIntervento : BaseCodeDescr
    {

        //Note_Valore_Numerico, Note_Valore_Stringa, Visibile

        public decimal note_Valore_Numerico { get; set; }

        public string note_Valore_Stringa { get; set; }

        public int visibile { get; set; }

        public NoteInterventoGruppi noteInterventoGruppi { get; set; }

        public NoteIntervento(int codice, int gruppo) : base(codice, "") 
        {
            noteInterventoGruppi = new NoteInterventoGruppi(gruppo);
        }
        public NoteIntervento(int codice) : base(codice, "") { }
        public NoteIntervento(int codice, string descrizione) : base(codice, descrizione) { }
        public NoteIntervento() : base() { }
    }
}
